tests/lib/enableRendererCoverage.js

```diff
@@ -0,0 +1,273 @@
+/**
+ * Coverage collecting for all BrowserWindows
```

Comment:
Maybe the name of the file (instead of a verb) should be changed to something like "rendererCoverageCollector" or similar?